classes/class-uagb-rest-api.php

```diff
@@ -90,24 +90,30 @@ public function content_pre_render( $block ) {
 			if ( empty( $block_css_style ) ) {
 				return $block;
 			}
+		
+			if ( ! empty( $block['attrs'] ) && is_array( $block['attrs'] ) ) {
```

Comment:
This logic should be mered with above code with || and retrun easrly
```suggestion
			if ( empty( $block_css_style ) || empty( $block['attrs'] ) || ! is_array( $block['attrs'] ) ) { 
			    return $block
			}
```

Reply:
Merged and returns early.